### PR TITLE
bump up kepler-action to 0.0.8

### PR DESCRIPTION
**Pull request by SamYuan1990**
bump up kepler-action to 0.0.8

### CODE REVIEW

**Comment by SamYuan1990** [comment]
@sunya-ch , I have not get the point https://github.com/sustainable-computing-io/local-dev-cluster/blob/main/main.sh#L93 what's the reason for mkdir -p report an error...

File: model_training/script.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ export PIPELINE_PREFIX=${PIPELINE_PREFIX-"std_"}
 export DATAPATH=${DATAPATH-"$(pwd)/data"}
 export ENTRYPOINT_IMG=${ENTRYPOINT_IMG-"quay.io/sustainable_computing_io/kepler_model_server:v0.7"}
 export MODEL_PATH=$DATAPATH
-export KUBECONFIG="/tmp/kubeconfig"
+export KUBECONFIG=${KUBECONFIG:"/tmp/kubeconfig"}
```
Comment (sthaha) [inline model_training/script.sh:26]:
```suggestion
export KUBECONFIG=${KUBECONFIG:-"/tmp/kubeconfig"}
```

File: model_training/script.sh [hunk 2]
```diff
@@ -23,7 +23,7 @@ export PIPELINE_PREFIX=${PIPELINE_PREFIX-"std_"}
 export DATAPATH=${DATAPATH-"$(pwd)/data"}
 export ENTRYPOINT_IMG=${ENTRYPOINT_IMG-"quay.io/sustainable_computing_io/kepler_model_server:v0.7"}
```
Comment (sthaha) [inline model_training/script.sh:24]:
```suggestion
export ENTRYPOINT_IMG=${ENTRYPOINT_IMG:-"quay.io/sustainable_computing_io/kepler_model_server:v0.7"}
```

**Review by sthaha — CHANGES_REQUESTED** [review]
Looks good overall, minor corrections though.

**Comment by sunya-ch** [comment]
@SamYuan1990 Could you please rebase this PR and resolve the change request by @sthaha ?

**Comment by SamYuan1990** [comment]
updated. @sunya-ch , @sthaha

**Review by sthaha — APPROVED** [review]

**Pull request merged**